pypeit/reduce.py

```diff
@@ -575,24 +584,26 @@ def find_objects(self, image, std_trace=None,
         """
 
         # Positive image
-        parse_manual = self.parse_manual_dict(manual_extract_dict, neg=False)
+        manual_dict= self.manual.dict_for_objfind(neg=False) if self.manual is not None else None
```

Comment:
Instead of having to do this conversion, can you instead construct the `ManualExtractionObj` class in such a way that you can pass it directly to `find_objects_pypeline`?

Reply:
well, I could instantiate it twice, but this seems cleaner.

Reply:
ok, did some refactoring.  Is cleaner and clearer now, I think.